js/widgets/forms/flipswitch.js

```diff
@@ -7,6 +7,7 @@
 
 define( [
 	"jquery",
+	"../../navigation/path",
```

Comment:
this should not depend on navigation for a simple regex just add the regex outside the widget to avoid multiple compilations.